Dockerfile

```diff
@@ -124,15 +124,14 @@ ENV RUBY_GEMS \
  jumbo-jekyll-theme:1.4.4 \
  # Used by ?
  jumbo-jekyll-theme:3.9.4 \
- # Used by ArmNN
- jumbo-jekyll-theme:4.4.4 \
- # Used by devicetree, TrustedFirmware
- jumbo-jekyll-theme:4.5.0 \
- # Used by linaro.org
- jumbo-jekyll-theme:4.7.7 \
- # Used by connect, ai
```

Comment:
To which sites did this refer? [connect.linaro.org](https://connect.linaro.org) and [www.96boards.ai](https://www.96boards.ai)?

Reply:
connect = connect.linaro.org 
ai = 96boards.ai

Reply:
Let's use the full names to ease discovery :smile: